android/assets/jsons/Civ V - Gods & Kings/TileImprovements.json

```diff
@@ -176,8 +176,17 @@
 		"uniques": ["Can be built outside your borders"],
 		"shortcutKey": "."
 	},
+    {
+        "name": "Repair",
+        "terrainsCanBeBuiltOn": ["Land"],
+        "turnsToBuild": 2,
+        "uniques": ["Can be built outside your borders"],
+        "shortcutKey": "R",
```

Comment:
That's already the shortcut for Road, so if you have a road and a pillaged improvement there's a conflict...

Reply:
Fixed